index.html

```diff
@@ -260,6 +260,439 @@ <h1 class="primary-text1">
+                <div class="card-block">
+                    <div class="card-header">
+                        <span class="card-header-text">Multi-Post Stories</span>
+                        <div class="card-header-frame">
+                            <div class="card-header-frame-client">
+                                <span class="card-header-frame-client-text">CANOPY</span>
+                            </div>
+                            <span class="counter">
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                                </svg>
+                            </span>
+                            <div class="card-header-frame-role">
+                                <span class="card-header-frame-role-text">Back End Dev</span>
+                            </div>
+                            <span class="counter">
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                                </svg>
+                            </span>
+                            <div class="card-header-frame-year">
+                                <span class="card-header-frame-year-text">2015</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-description"> A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                    <ul class="card-tags">
+                        <li class="card-tag"><span class="card-tag-text">html</span></li>
+                        <li class="card-tag"><span class="card-tag-text">css</span></li>
+                        <li class="card-tag"><span class="card-tag-text">javaScript</span></li>
+                    </ul>
+                    <a class="action" href="#" target="_blank">
+                        <button type="button" class="button-link"><span class="button-link-text">See Project</span></button>
+                    </a>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <section class="about-me">
+        <div class="primary-text-container about-me-top-section">
+            <h2 class="primary-text1 about-me-header">About Myself</h2>
+            <p class="primary-text2">Hello I&#39m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don&#39t hestiate to contact me.</p>
+            <a class="about-me-button" href="#" target="_blank">
+                <button type="button" class="button-link"><span class="button-link-text">Get my resume</span></button>
+            </a>
```

Comment:
- [ ] A "Let's connect" title is missing in your design. Please add this title to follow the Figma design and not miss any element.
 - [ ] This 'Get my resume' button should be in 'button' or an 'anchor' tag. There is no need to wrap a `<button>` tag around an `<a>` tag. Please remove the `<a>` tag around the button.
 -Your design: 
<img width="378" alt="image" src="https://user-images.githubusercontent.com/61919617/191732331-b5b413e1-0ad7-4544-987a-e7005b89daf7.png">

 -Figma design:
<img width="336" alt="image" src="https://user-images.githubusercontent.com/61919617/191732421-22101b7c-bbb5-47d7-9015-901401ac37f6.png">